.env

```diff
@@ -1,3 +1,3 @@
-# REACT_APP_EGGHEAD_BASE_URL=https://egghead-io-staging.com
-REACT_APP_EGGHEAD_BASE_URL=http://egghead.dev:5000
+REACT_APP_EGGHEAD_BASE_URL=https://egghead-io-staging.com
```

Comment:
I keep this on staging for contributors who clone without setting up local dev; so they can just `yarn`, `yarn dev`, log in with a staging account and they are good to contribute.